dotCMS/src/main/java/com/dotmarketing/portlets/cmsmaintenance/ajax/CMSMaintenanceAjax.java

```diff
@@ -313,6 +346,25 @@ public String doBackupExport(String action, boolean dataOnly) throws IOException
 			f.mkdirs();
 			deleteTempFiles();
 			String message = "";
+		if(action.equals("createZip")) {
+			if (!dataOnly) {
+				moveAssetsToBackupDir();
+			}
+			message = "Creating XML Files. ";
+			createXMLFiles();
+			String x = UtilMethods.dateToJDBC(new Date()).replace(':', '-').replace(' ', '_');

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use the Java 8 Date and Time API instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2143)